src/objects/avatar/util/getAvatarDrawDefinition.ts

```diff
@@ -32,16 +32,27 @@ const basePartSet = new Set<AvatarFigurePartType>([
   AvatarFigurePartType.Head,
 ]);
 
+const headComponents = ["hd",
+  "fc",
+  "ey",
+  "hr",
+  "hrb",
+  "fa",
+  "ea",
+  "ha",
+  "he",];
+
```

Comment:
Probably change it to a `Set` aswell

Reply:
Unsure about set, accessing at certain indexes etc for where this is used?